src/test/java/dog/pawbook/logic/commands/AddDogCommandTest.java

```diff
@@ -2,13 +2,13 @@
 
 import static dog.pawbook.testutil.Assert.assertThrows;
 import static java.util.Objects.requireNonNull;
-import static org.junit.jupiter.api.Assertions.assertEquals;
+//import static org.junit.jupiter.api.Assertions.assertEquals;
```

Comment:
Refrain from commenting like that, a better practice when using git is to simply delete all these lines that you don't need or want to disable, commit this set of changes as an individual commit. If you intend to fix it later, maybe leave a todo around the original area. This way even if we don't implement it there won't be a need to clean up the code much. But if you do intend to restore this portion, it will be simply a matter of finding the commit that removed the lines and reverting it, creating an additional commit that essentially negates the removal yet retaining full history.

Reply:
I see, ok will remove these lines!